docs/useRequireAccess.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: "useRequireAccess"
+---
+
+# `useRequireAccess`
+
+This hook calls the `authProvider.canAccess()` method on mount for a provided resource and action (and optionally a record). If users don't have access, it will redirect them to the [`/unauthorized`](./Admin.md#unauthorized) page. If `authProvider.canAccess()` throws an error, it will redirect them to the [`/authentication-error`](./Admin.md#authenticationerror) page.
+
+## Usage
+
+`useRequireAccess` takes an object `{ action, resource, record }` as argument. It returns an object describing the state of the request. As calls to the `authProvider` are asynchronous, the hook returns a `isPending` state in addition to the `canAccess` key.
```

Comment:
```suggestion
`useRequireAccess` takes an object `{ action, resource, record }` as argument. It returns an object describing the state of the request. As calls to the `authProvider` are asynchronous, the hook returns a `isPending` state.
```

We never care about the actual `canAccess` as it can't be false.